src/web_app/src/pages/run.tsx

```diff
@@ -206,6 +156,11 @@ export function RunPage() {
                 Suivant
                     </Button>
                 </Stack>
+                { open &&
+            <Alert onClose={handleClose} severity="warning" sx={{width: "100%"}}>
+            Le solveur est actuellement entrain de tourner. En cliquant sur
+            Suivant, vous redémarrez une nouvelle résolution.
```

Comment:
well if you try to run a new resolution the server will answer as stated in the documentation but won't start any new resolution unless the previous one is done ^^.